opentelemetry_collector/components.go

```diff
@@ -27,30 +28,37 @@ import (
 	"github.com/googlecloudplatform/appengine-sidecars-docker/opentelemetry_collector/receiver/vmagereceiver"
 )
 
+func appendError(err error, newerr error) error {
+	if err != nil {
```

Comment:
Should this condition be inverted?
i.e. if the existing error is nil, then you want to return the new error. And if the existing error is not nil, then you want to wrap it with the new error?

Reply:
Thank you! I updated the function.